ci/python_files_and_folders.py

```diff
@@ -0,0 +1,13 @@
+"""Determine the Python files and folders in the current directory."""
+
+from pathlib import Path
+
+def python_files_and_folders() -> list[str]:
+    """Return the Python files and folders in the current directory."""
+    python_files = [python_file.name for python_file in Path(".").glob('*.py') if not python_file.name.startswith(".")]
+    python_folders = [folder_name for folder_name in ("src", "tests") if Path(folder_name).exists()]
```

Comment:
Hmm, this is a bit sneaky perhaps

Reply:
I guess looking up the folders could be made more generic by looking for folders with python files in it, but I didn't feel it was worth it.

The same could be done in bash of course, but less readable :-)